actionpack/lib/action_dispatch/middleware/templates/rescues/invalid_statement.text.erb

```diff
@@ -6,6 +6,8 @@
 <%= @exception.message %>
 <% if defined?(ActiveStorage) && @exception.message.match?(%r{#{ActiveStorage::Blob.table_name}|#{ActiveStorage::Attachment.table_name}}) %>
 To resolve this issue run: rails active_storage:install
+<% if defined?(ActionMailbox) && @exception.message.match?(%r{#{ActionMailbox::InboundEmail.table_name}}) %>
+  To resolve this issue run: rails action_mailbox:install
```

Comment:
Probably needs to be outdented like the line above.

Reply:
Updated!